src/constructor.io/client/modules/Catalog.cs

```diff
@@ -380,5 +381,504 @@ public async Task<string> DeleteItemGroups(ItemGroupsRequest itemGroupsRequest)
 
             throw new ConstructorException("DeleteItemGroups response data is malformed");
         }
+
+        // Facets
+        internal string CreateFacetUrl([Optional] string section, [Optional] string facetGroup, [Optional] Hashtable queryParams)
```

Comment:
not a fan of the [Optional] syntax, I'd rather we stay consistent and do string section = "Products" unless this has some benefits

Reply:
Cool, makes sense. I'll make the changes :D